login.html

```diff
@@ -1,46 +1,42 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="login.css">
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="login.css" />
     <title>login</title>
-</head>
-<body>
+  </head>
+  <body>
     <div class="logo_box">
-        <img class="logo" src="images/login/logo.png">
-        <div class="login_box">
-            <div id="email">
-                <span class="email">이메일</span>
-                <h2 class="email_input"></h2>
-            </div>
-            <div id="password">
-                <span class="password">비밀번호</span>
-                <h2 class="password_input"></h2>
-            </div>
-            <h3 class="login_button">
-                로그인
-            </h3>
-            <div id="easy_login">
-                <div class="easy_login_box">
-                    <h3 class="easy_login">간편 로그인하기</h3>   
-                    <div class="login_sns">
-                        <a href="https://www.google.com/"
-                            target="_blank">
-                            <img class="login_google" src="images/login/google.png">
-                        </a>
-                            <a href="https://www.kakaocorp.com/page/"
-                            target="_blank">
-                            <img class="login_kakao" src="images/login/kakao.png">
-                        </a>
-                    </div>
-                </div>
-            </div>
-            <div class="signup">
-                <p class="signup_ask">판다마켓이 처음이신가요?</p>
-                <a class="signup_link" href="signup.html">회원가입</a>
+      <img class="logo" src="images/login/logo.png" />
+      <div class="login_box">
+        <div id="email_password">
+          <span class="email_password">이메일</span>
+          <div class="email_password_input"></div>
+        </div>
+        <div id="email_password">
+          <span class="email_password">비밀번호</span>
+          <div class="email_password_input"></div>
```

Comment:
❗️ 수정요청
유저의 값을 받아야하니 해당 요소는 클래스명처럼 input이 더 적절할 것 같습니다.

```suggestion
          <input type="text" class="email_password_input"></input>
```